src/main/java/org/jointheleague/modules/MoreThan7LettersListener.java

```diff
@@ -0,0 +1,58 @@
+package org.jointheleague.modules;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.javacord.api.entity.message.Message;
+import org.javacord.api.event.message.MessageCreateEvent;
+
+import net.aksingh.owmjapis.api.APIException;
+
+public class MoreThan7LettersListener extends CustomMessageCreateListener{
+
+	public MoreThan7LettersListener(String channelName) {
+		super(channelName);
+	}
+
+	@Override
+	public void handle(MessageCreateEvent event) throws APIException {
+		if (event.getMessageAuthor().asUser().get().isBot()){
+			return; //stops the bot from reading its own messages
+		}
+		//get all the words of their message.
+		String alphabet = "abcdefghijklmnopqrstuvwxyz";
+		String[] words = event.getMessageContent() //gets the message
+				.replaceAll("[^ a-zA-Z0-9]", "") //removes all punctuation
+				.replaceAll("  ", " ")//removes double spaces
+				.split(" ");
+		//gets longest word
+		String longestWord = getFirstLongWord(words); //if there is no word longer than 7 letters, it will return "."
+		if (longestWord.contentEquals("."))
```

Comment:
Instead of checking for the specific String of ".", just check if the returned string is less than 7. This way, the user of the function doesn't have to know all the details about the function to use it.